sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Breaking Changes
 
+- Cookies are no longer set on requests by default. They can be re-enabled for `HttpClientTransport` using the `Azure.Core.Pipeline.HttpClientTransport.EnableCookies` switch, or the `AZURE_CORE_HTTPCLIENT_ENABLE_COOKIES` environment variable.
```

Comment:
nit: Here is a great example that provides a bit more context about app compat switches

https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/CHANGELOG.md#new-features-2